File: modulo1/segmento3/apunte8 "Vectores y Listas"/ejercicio3/Ejercicio3.java
import java.util.Scanner;
import  java.util.ArrayList;

public class Ejercicio3{
    public static void main(String[] args) {
        Scanner teclado = new Scanner(System.in);
        ArrayList <Integer>lista1 = new ArrayList<Integer>();

        for(int i=0; i<5;i++)
        {
            System.out.println("Ingrese 5 numeros para multiplicar por 2: ");
            Integer numero = Integer.parseInt(teclado.nextLine());
            lista1.add(numero);
        }
        System.out.println(lista1);

        ArrayList<Integer>lista2 = new ArrayList<Integer>();
        for(int i=0; i<5;i++)
        {
            lista2.add(lista1.get(i)*2);
        }
        System.out.println("Los numeros ingresados multiplicado por 2, dan como resultado: " + lista2);
    }
}

File: modulo1/segmento3/apunte8 "Vectores y Listas"/ejercicio3/Ejercicio3.py
lista1 = []
lista2 = []

for i in range(5):
    numero = int(input("Ingrese 5 numeros para multiplicar por 2: "))
    lista1.append(numero)
print(lista1)

for i in range(len(lista1)):
    lista2.append(lista1[i]*2)
print("Los numeros ingresado multiplicado por 2, dan como resultado: ", lista2)
